modules/network-observability-enriched-flows-kafka.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// network_observability/understanding-network-observability-operator.adoc
+
+:_content-type: Procedure
+[id="network-observability-enriched-flows-kafka_{context}"]
+= Export enriched network flow data
+
+You can send network flows to Kafka, so that they can be consumed by any processor or storage that supports Kafka input, such as Splunk, Elastic, or Fluentd,
```

Comment:
As you've found out it's ElasticSearch (the product), not Elastic (the company)

Reply:
Aw dangit! I thought I changed this after I asked you about it. Thanks for noting it.